cloudformation/elasticsearch.yaml

```diff
@@ -127,4 +157,11 @@ Resources:
               Action: 'es:*'
               Resource:
                 - !Sub 'arn:aws:es:${AWS::Region}:${AWS::AccountId}:domain/*'
+            - Action: es:*
+              Principal:
+                AWS:
+                  - !Join ["/", [ !Sub 'arn:aws:sts::${AWS::AccountId}:assumed-role', !Ref KibanaCognitoRole, CognitoIdentityCredentials]]

```

Comment:
When defining ARNs, its better to use `${AWS::Partition}` instead of hardcoding `aws` because special regions like GovCloud or China have different partition names (i.e. aws-gov, aws-cn, etc). 

```
arn:${AWS::Partition}:sts::${AWS::AccountId}:assumed-role
```